nss-cohort-37/Bangazon-Site-Blue-Barracudas
# Nm shopping cart

# Description

Shopping cart view and adding items to shopping cart product types/details page. Complete order and cancel order. 

https://github.com/nss-cohort-37/Bangazon-Site-Blue-Barracudas/issues/13
https://github.com/nss-cohort-37/Bangazon-Site-Blue-Barracudas/issues/36
https://github.com/nss-cohort-37/Bangazon-Site-Blue-Barracudas/issues/9
https://github.com/nss-cohort-37/Bangazon-Site-Blue-Barracudas/issues/2
https://github.com/nss-cohort-37/Bangazon-Site-Blue-Barracudas/issues/6

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions
1) First you will need to DELETE your database and do and Update Database as we changed the restricted feature to cascade 
2) Add products to your shopping cart from the product types details. 
3) Click view cart. You should see your open shopping cart, you can delete products from the cart
4) Complete order and entire payment type 
5) Cancel order and your shopping cart will be empty. 



# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
